charts/oam-kubernetes-runtime/templates/webhook.yaml

```diff
@@ -0,0 +1,43 @@
+{{- if .Values.useWebhook -}}
+---
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: {{ include "oam-kubernetes-runtime.fullname" . }}
+  labels:
+    {{- include "oam-kubernetes-runtime.selectorLabels" . | nindent 4 }}
+webhooks:
+  - name: "validate.applicationconfigurations.core.oam.dev"
+    rules:
+      - apiGroups:   ["core.oam.dev"]
+        apiVersions: ["v1alpha2"]
+        operations:  ["CREATE", "UPDATE","DELETE"]
+        resources:   ["applicationconfigurations"]
+        scope:       "Namespaced"
+    clientConfig:
+      service:
+        namespace: {{.Release.Namespace}}
+        name: {{ template "oam-kubernetes-runtime.name" . }}-webhook
+        path: /validating-applicationconfigurations
```

Comment:
is there any way to make sure this path is the same as the path in the below function

func Register(server *webhook.Server) {
	server.Register("/validating-applicationconfigurations", &webhook.Admission{Handler: &ValidatingHandler{}})
}

Reply:
No other way, we can configure it right. It's configurable here